aspnetcore/security/enforcing-ssl.md

```diff
@@ -161,7 +194,7 @@ ASP.NET Core 2.1 or later implements HSTS with the `UseHsts` extension method. T
 
 `UseHsts` isn't recommended in development because the HSTS settings are highly cacheable by browsers. By default, `UseHsts` excludes the local loopback address.
 
-For production environments implementing HTTPS for the first time, set the initial HSTS value to a small value. Set the value from hours to no more than a single day in case you need to revert the HTTPS infrastructure to HTTP. After you're confident in the sustainability of the HTTPS configuration, increase the HSTS max-age value; a commonly used value is one year. 
+For production environments implementing HTTPS for the first time, set the initial HSTS value to a small value. Set the value from hours to no more than a single day in case you need to revert the HTTPS infrastructure to HTTP. After you're confident in the sustainability of the HTTPS configuration, increase the HSTS max-age value; a commonly used value is one year.
```

Comment:
Be specific about the option to set (HstsOptions.[MaxAge](https://github.com/aspnet/BasicMiddleware/blob/1e84159abb0f364bd1716c96992db2cd857d663f/src/Microsoft.AspNetCore.HttpsPolicy/HstsOptions.cs#L21)).